language-examples/python/vectara-rest/rest_create_corpus.py

```diff
@@ -40,4 +40,10 @@ def create_corpus(customer_id: int, admin_address: str, jwt_token: str):
                        response.reason,
                        response.text)
         return response, False
-    return response, True
\ No newline at end of file
+
+    message = response.json()
+    if message["status"]["code"] != "OK":
```

Comment:
Should there be a check that message["status"] is not None?

Reply:
noting that the REST delete document also does not set it when successful. I think these two cases are not consistent with the other current APIs.

Reply:
In general, the API (and even backend code) assumes that the absence of status (or empty since OK=0) means OK. I think we should follow that everywhere in this example guide, even if this particular call is setting it in a slightly different fashion.

Reply:
> the API (and even backend code) assumes that the absence of status (or empty since OK=0) means OK.

In terms of what's easiest for the end-user, though, wouldn't it be easier if it was just consistently set to OK anytime the result was OK? That's the way I interpreted Laurence's comment.

Reply:
Most of our APIs set the top-level status to OK when the operation succeeds. There are just a couple of our APIs which don't do this, which can be confusing for users. Here's an example: a user using the file upload API for the first time seeing an empty status returned: "response status empty, what the hell's that?" at 43:45 in episode 10 here: https://twitter.com/gputopia/status/1723203092647137636?s=46&t=Yio6vrQ3sj6F2iOmf6GQ4A.

I would prefer avoid adding a lot of boilerplate code that protects against impossible conditions, but if we want this pattern to be used in our examples to guard against API inconsistencies, we can do that. The next generation APIs should allow cleaner code.

Reply:
Done.